.github/workflows/test.yml

```
@@ -21,6 +21,14 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Remove msys64
+      if: runner.os == 'Windows'
+      run: Remove-Item -LiteralPath "C:\msys64\" -Force -Recurse
+
+    - name: Install dependencies
+      if: runner.os == 'Windows'
+      run: choco install llvm -y
```

On wallet.rs we found that downloading the prebuilt LLVM from the LLVM repo was significantly faster (<30s):

```suggestion
    - name: Install LLVM and Clang (Windows)
      uses: KyleMayes/install-llvm-action@32c4866ebb71e0949e8833eb49beeebed48532bd
      if: matrix.os == 'windows-latest'
      with:
        version: "11.0"
        directory: ${{ runner.temp }}/llvm
      
    - name: Set LIBCLANG_PATH (Windows)
      run: echo "LIBCLANG_PATH=$((gcm clang).source -replace "clang.exe")" >> $env:GITHUB_ENV
      if: matrix.os == 'windows-latest'
```

---

Thanks